### DISCUSSION TITLE
High-level specifications for iCub & iCub stand

### DISCUSSION THREAD

**Discussion by frank-foerster** [robotology/community#408] [Q&A] [answered]
## Brief description
We will be "moving house" at some point and for that matter, I need to know some high-level technical specs
of both the iCub itself, as well as of its stand. I have been searching around a bit, but could not find all necessary information, so hence the question here. 

## Detailed context
I need to know the spatial dimensions (height, width, breadth), and the weight of both the iCub robot and its stand.
In the following I will state what I found, question marks indicate that I could not find this information.
And given that this is for deciding a place for robot, `width', and `breadth' should possibly refer to the maximum extension
of the robot arms (imagine a ballerina stretching the arms out, and twirling around its own axis)

_iCub_:
**_weight_**: 25kg 
(according to [> this](https://robots.ieee.org/robots/icub/) outside source. Please correct if this is wrong)

**_height_**: ?

**_width_**: ?
(assume maximally stretched out arms)

**_breadth_**: ?
(I would assume the same as the width, assuming the ballerina scenario mentioned above)

_iCub stand_:
**_weight_**: ?

**_height_**: ?
(for height I would assume the height including the length/height of the pole to which the iCub is attached)

**_width_**: 91cm 

**_breadth_**: 80cm
(I took or estimated width/breadth based on a design PDF on the IIT server, a link to which I can't find at the moment)

Many thanks for any help.

**Comment by pattacini** [comment]
Hi @frank-foerster 

We'll give you this info soon. What is the HW version of your iCub? Does it have the battery backpack (probably not)?

**Comment by frank-foerster** [comment]
Hi @pattacini ,

it doesn't have a backpack. It has been upgraded 5 years ago or so, so I'm not totally sure what verison it counts as now.
Thanks

**Comment by pattacini** [comment]
Can you look it up in https://icub-tech-iit.github.io/documentation/icub_versions, although knowing it won't change that much in terms of dimensions and weights?

**Comment by pattacini** [comment]
@frank-foerster we're going to give you precise dimensions straight from the CAD.
Concerning the weights instead, it'd be better off referring to https://github.com/robotology/QA/issues/61 where we took measurements for a bunch of representative robots using the scale.

In particular, your robot should be considered being of the same category of `iCubGenova01` ➡ `30.3 kg`.

**Comment by salvi-mattia** [comment, marked as answer]
I hope it can help [all measure in mm]

![immagine](https://user-images.githubusercontent.com/43994292/86117437-55480000-bacf-11ea-8079-04b6ebe4860b.png)
****
I don't know wich kind of trolley you are using, this is the most common one

![immagine](https://user-images.githubusercontent.com/43994292/86118139-88d75a00-bad0-11ea-927c-4bec07f5c1a0.png)

**Comment by frank-foerster** [comment]
Hi @pattacini and @salvi-mattia ,

thank you very much for the excellent information. Judging by the list that Ugo linked, I think we have version 1.4. We do have the 2nd version of the head, and  version 2.0 seems to include another head-related improvement. However, our head was upgraded either before we got the skin sensors or at the same time if I remember correctly. So I think our "head upgrade" is not the same as the one that is part of the v2 upgrade. 

But in terms of weight and dimensions the information you provided and linked here is more than sufficient. 

Many thanks!
